content/en/docs/tasks/administer-cluster/kubeadm/upgrading-linux-nodes.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Upgrading Linux nodes
+content_type: task
+weight: 50
```

Comment:
OK, but please bump `content/en/docs/tasks/administer-cluster/kubeadm/upgrading-windows-nodes.md` to a number larger than 50.

Reply:
Thanks for looking at this.
I will update the weight for these pages.

Reply:
Addressed the comment, PTAL!